feat(account): close and erase account (#2991)
scaleway/docs-content: console/account/how-to/manage-my-personal-data-gdpr.mdx

## How to erase your account and personal data

If you do not want to use Scaleway services anymore, you can [close your account](/console/account/how-to/close-account/) and/or delete your personal data.

> ### Closing your account or erasing your personal data: what’s the difference

Reviewer: ```suggestion
### Closing your account or erasing your personal data: what’s the difference?
```

<Macro id="close-vs-erase" />

### How to erase your personal data

Erasing your account and personal data will permanently delete your Organization’s servers, resources, services, backups, and all data related to your account. 

<Macro id="requirements" />
- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Ownership](/identity-and-access-management/iam/concepts/#owner) of the Organization you intend to delete

<Message type="important">
  Before requesting your personal data be deleted, ensure all your invoices are paid, and no resources are running. **Your request to erase your account will fail if you have unpaid consumption.** 
</Message>

1. Click your Organization’s name at the top-right corner of the Scaleway console. 
2. Click **Profile** to access your User Account space. 
3. Click the **My privacy** tab 
4. Scroll down to the **Erase my personal data** panel.

> 5. Click the **Send request** button once you read and agree with the warning message.

Reviewer 2: ```suggestion
5. Click the **Send request** button once you have read and agreed with the warning message.
```

6. Type **Erase**, then click **Send request** to confirm your decision.

> 7. A message appears to confirm that your account and personal data will be erased soon. The process can take up to 72 hours.

Reviewer 2: ```suggestion

A message appears to confirm that your account and personal data will be erased soon. The process can take up to 72 hours.
```

Reviewer 2: removed the step because it is not really a step

## How to erase your account and personal data

If you do not want to use Scaleway services anymore, you can [close your account](/console/account/how-to/close-account/) and/or delete your personal data.

### Closing your account or erasing your personal data: what’s the difference?

<Macro id="close-vs-erase" />

### How to erase your personal data

Erasing your account and personal data will permanently delete your Organization’s servers, resources, services, backups, and all data related to your account. 

> <Macro id="requirements" />

Reviewer 3: ```suggestion
<Macro id="requirements" />

```
